november+december.plan

```diff
@@ -3,8 +3,6 @@
 - example: json
 - example: testing
 - example: reading and writing files
-- example: switch
-- example: variables
 - example: mutable and immutable
 - example: operators (complete it)
 - example: pointers
```

Comment:
You can remove line `19` because GitBook is updating everything once we merge anything to the master branch.